src/components/vistas/home/index.js

```diff
@@ -21,7 +22,7 @@ function Home() {
             QUIERO DONAR
             </button>
           <Link to="/medicine-selector">
-            {} {/*En esta linea va a ir el componente que le sigue*/}
+            {MedicineSelector}
```

Comment:
Procura usar la notación JSX cuando se pueda.
```suggestion
            <MedicineSelector />
```

Reply:
Estoy importando el componente siguiente, si uso la notacion ' <MedicineSelector /> se me pinta abajo !